sdk_extensions/zpages/src/main/java/io/opentelemetry/sdk/extensions/zpages/TracezZPageHandler.java

```diff
@@ -486,18 +486,19 @@ private void emitHtmlBody(Map<String, String> queryMap, PrintStream out)
               // Display latency based span
               LatencyBoundary latencyBoundary = LatencyBoundary.values()[subtype];
               spans =
-                  dataAggregator.getOkSpans(
-                      spanName,
-                      latencyBoundary.getLatencyLowerBound(),
-                      latencyBoundary.getLatencyUpperBound());
+                  new ArrayList<>(
```

Comment:
We may as well stick with immutable `ImmutableList.sortedCopyOf`. I guess one problem was that we didn't go "all-in" on `ImmutableList`, if we did then issues like sorting a collection become more obviously a bug. Let's try to keep the standard in this code that all the containers are immutable.

Reply:
So instead of `new ArrayList<>()`, I should use `ImmutableLIst.sortedCopyOf`?

Reply:
Yeah

Reply:
I will remove `new ArrayList<>()` and instead of `Collections.sort`, I will do `spans = ImmutableList.sortedCopyOf(comparator, spans)`.